frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/delete-block-pool-modal.tsx

```diff
@@ -86,7 +86,11 @@ const DeleteBlockPoolModal = withHandlePromise((props: DeleteBlockPoolModalProps
 
   // Delete block pool
   const deletePool = () => {
-    handlePromise(k8sKill(CephBlockPoolModel, blockPoolConfig), () => close());
+    handlePromise(k8sKill(CephBlockPoolModel, blockPoolConfig), () => {
+      close();
+      // eslint-disable-next-line no-restricted-globals
+      history.back();
```

Comment:
```suggestion
      history.back();
```
Import history from `@console/internal/components/utils/router`